ros_buildfarm/release_job.py

```diff
@@ -139,11 +139,14 @@ def configure_release_jobs(
     all_view_configs = {}
     all_job_configs = OrderedDict()
 
-    job_name, job_config = configure_import_package_job(
-        config_url, rosdistro_name, release_build_name,
-        config=config, build_file=build_file, jenkins=jenkins, dry_run=dry_run)
-    if not jenkins:
-        all_job_configs[job_name] = job_config
+    for os_name, _ in platforms:
+        if package_format_mapping[os_name] not in ('rpm',):
+            job_name, job_config = configure_import_package_job(
+                config_url, rosdistro_name, release_build_name,
+                config=config, build_file=build_file, jenkins=jenkins, dry_run=dry_run)
+            if not jenkins:
+                all_job_configs[job_name] = job_config
+            break
```

Comment:
The `break` jumps out at me here. I don't foresee us doing so, but would this prevent a release configuration which mixes rpm and deb packages from properly generating import package jobs depending on the order of iteration?

Reply:
The `break` happens when a non-RPM platform is specified, and RPM platforms are ignored. The order shouldn't matter here - as long as at least 1 specified platform is non-RPM, you'll get an import job.